Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md

```diff
@@ -61,12 +61,17 @@ By default, the following Field Types are available:
 - **Single Choice**:
     Displays a list of items with a radio button for each item where the user can select a single option.
 
-    ![singlechoice](images/singlechoice-v9.png)
+    ![Single choice](images/singlechoice-v9.png)
 
 - **Title and Description**:
-    Displays a title and description that are set as default values.
+    Displays a read-only title and description for a set of form fields.
 
-    ![Radiobuttonlist](images/titleanddescription-v9.png)
+    ![Title and description](images/titleanddescription-v9.png)
+
+- **Rich Text**:
+    Displays read-only formatted text that can be used to provide additional copy within a form.
```

Comment:
```suggestion
    Displays a read-only formatted text that can be used to provide an additional copy within a Form.
```

Reply:
As above.  Also, here I don't think we want the "an".  I'm looking to say here you can add additional copy in the sense of "additional formatted text" rather than a copy as in "a duplicate".

Reply:
Oh ok, now it makes sense to me 😃I was thinking of it as an additional duplicate copy. 

Can we maybe rephrase it to make it more clear? Maybe something like: 
```suggestion
    Displays a read-only tool that can be used to provide additional formatted text within a Form.
```